pkg/apis/messaging/v1/subscription_types.go

```diff
@@ -89,7 +88,7 @@ type SubscriptionSpec struct {
 	// channel, giving the user more control over what semantics should
 	// be used (drain the channel first, possibly have events dropped,
 	// etc.)
-	Channel corev1.ObjectReference `json:"channel"`
+	Channel duckv1.KReference `json:"channel"`
```

Comment:
Same here (well, this is probably where we need to change it. The comment needs an update to KReference.